kore/app/repl/Main.hs

```diff
@@ -242,10 +241,7 @@ mainWithOptions LocalOptions{execOptions, simplifierx} = do
 
                         SMT.runSMT
                             smtConfig
-                            ( give
-                                (MetadataTools.build validatedDefinition)
-                                (declareSMTLemmas validatedDefinition)
-                            )
+                            (declareSMTLemmas (MetadataTools.build validatedDefinition) validatedDefinition)
```

Comment:
Please break this line so it doesn't exceed 80 characters
https://github.com/runtimeverification/haskell-backend/wiki/Style-guide#line-length